components/lib/cascadeselect/CascadeSelectBase.js

```diff
@@ -160,7 +160,9 @@ export const CascadeSelectBase = ComponentBase.extend({
         tabIndex: null,
         transitionOptions: null,
         value: null,
-        children: undefined
+        children: undefined,
+        panelStyle: null,
+        panelClassName: null
```

Comment:
Just a nit i usually try and put these in alphabetical order with the exception of children i always leave at the bottom (no particular reason) :)

Reply:
Haha, totally got it :-)